signac/__main__.py

```diff
@@ -1468,9 +1469,17 @@ def main():
 
     parser_find = subparsers.add_parser(
         "find",
-        description="""All filter arguments may be provided either directly in JSON
-                       encoding or in a simplified form, e.g., -- $ signac find a 42 --
-                       is equivalent to -- $ signac find '{"a": 42}'.""",
+        formatter_class=FlexiFormatter,
```

Comment:
Can we just write the raw string we want to print? I don’t think adding a dependency is the right course of action if a literal expression of the desired string (with continuation symbols and explicit \n newlines) is viable.

Reply:
I don't think so.

I tried
```
description=r""" blah blah with\
trailing lines and \n etc"""``` and it gets line wrapped.

Reply:
This formatter is in the standard library and lets you manually write the string: https://docs.python.org/3/library/argparse.html#argparse.RawDescriptionHelpFormatter

I pushed a proposed change for this in b2efd854b40a2a7c44b7d8b89fd704582492b228.

Reply:
That works, thanks! I'd also rather not add a dependency but it seemed like it was about to get merged into Python.

I tried RawDescriptionHelpFormatter first but I didn't know about the Python automatic string continuation to have it wrap lines while preserving newlines.